src/Voxel.MiddyNet/MiddyNet.cs

```diff
@@ -70,6 +93,9 @@ private void InitialiseMiddyContext(ILambdaContext context)
             }
 
             MiddyContext.AdditionalContext.Clear(); //  Given that the instance is reused, we need to clean the dictionary.
+            MiddyContext.MiddlewareBeforeExceptions = new List<Exception>();
+            MiddyContext.HandlerException = null;
+            MiddyContext.MiddlewareAfterExceptions = null;
```

Comment:
Why null and not new List or Clear?

Reply:
Because I want to prevent users from messing with a collection that has no value in this context.